app/scripts/components/common/catalog/utils.ts

```diff
@@ -1,9 +1,15 @@
 import { omit, set } from 'lodash';
+import { optionAll } from '$components/common/browse-controls/constant';
 
 export enum CatalogActions {
+  TAXONOMY_MULTISELECT = 'taxonomy_multiselect',
   CLEAR = 'clear',
```

Comment:
I think we can get rid of the CLEAR option if we are going to be explicit in the about which CLEAR operation we are doing (taxonomies or search)

Reply:
I think it can go either way, but `CLEAR` might be a better option when we intend to reset the state of filtering? I changed the modal to use `CLEAR` whent it needs to reset the state!